cluster/ubuntu/util.sh

```diff
@@ -364,6 +369,11 @@ function provision-master() {
     ubuntu/binaries/master/ \
     "${MASTER}:~/kube"
 
+  # copy the flannel binaries and scripts, if flannel is being used
```

Comment:
Previously the copy of the Flannel files was unconditional and did not require distinct commands because it was a matter of directory contents.  I split it out for the case when Flannel is not used.  I would be willing to always copy the Flannel files to the remote `~/kube`, that itself is harmless (merely unnecessary in the CNI case).